src/emissions-workspace/emissions-checks.service.ts

```diff
@@ -0,0 +1,28 @@
+import { BadRequestException, HttpStatus, Injectable } from '@nestjs/common';
+import { LoggingException } from '@us-epa-camd/easey-common/exceptions';
+import { Logger } from '@us-epa-camd/easey-common/logger';
+import { EmissionsImportDTO } from '../dto/emissions.dto';
+import { WeeklyTestSummaryCheckService } from '../weekly-test-summary-workspace/weekly-test-summary-check.service';
+
+@Injectable()
+export class EmissionsChecksService {
+  constructor(
+    private readonly logger: Logger,
+    private readonly weeklyTestSummaryCheckService: WeeklyTestSummaryCheckService,
+  ) {}
+  private throwIfErrors(errorList: string[]) {
+    if (errorList.length > 0) {
+      throw new LoggingException(errorList.toString(), HttpStatus.BAD_REQUEST);
+    }
+  }
+
+  runChecks(payload: EmissionsImportDTO) {
+
+    const errorList: string[] = [];
+    const errors = this.weeklyTestSummaryCheckService.runChecks(payload);
```

Comment:
Ah, that const, was meant to be a let. The idea was to have one errors variable and set it to w.e check is running. After check finishes it would get pushed into errorList. Though I think I like the idea of just naming them with specific names and pushing everything to errorList at the end so I'll go with that.